Base/usr/share/man/man1/Shell.md

```diff
@@ -5,9 +5,10 @@
 ## Synopsis
 
 ```**sh
-$ Shell [--skip-shellrc]
+$ Shell [--skip-shellrc] [--live-formatting]
```

Comment:
I added `--live-formatting` here as this is the invocation for REPL mode, and the option makes sense there.